File: C1.UWP/CS/BasicLibrarySamples/View/TreeView/DragDrop/Department.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BasicLibrarySamples
{
    /// <summary>
    /// Business object used to represent a Department.
    /// </summary>
    public class Department
    {
        public Department()
        {
            Employees = new ObservableCollection<Employee>();
        }

        public int DepartmentID { get; set; }
        public string Name { get; set; }
        public string GroupName { get; set; }
        public ObservableCollection<Employee> Employees { get; set; }
    }
}

File: C1.UWP/VB/BasicLibrarySamples/View/TreeView/DragDrop/Department.vb
Imports System.Threading.Tasks
Imports System.Text
Imports System.Linq
Imports System.Collections.ObjectModel
Imports System.Collections.Generic
Imports System

''' <summary>
''' Business object used to represent a Department.
''' </summary>
Public Class Department
    Public Sub New()
        Employees = New ObservableCollection(Of Employee)()
    End Sub

    Public Property DepartmentID() As Integer
    Public Property Name() As String
    Public Property GroupName() As String
    Public Property Employees() As ObservableCollection(Of Employee)
End Class